### ISSUE TITLE
Allow data: URIs

### ISSUE THREAD

**Issue by gibson042** [bterlson/ecmarkup#125]
`href` attributes do not allow URIs with the "data" scheme because [the CLI sets `fetch` to a thin wrapper around `fs.readFile`](https://github.com/bterlson/ecmarkup/blob/49471301cdb02fbff2c999c655110f1a1164a483/src/cli.ts#L25). It would be convenient and expected for such values to be allowed (e.g., it would have saved me a file in https://github.com/gibson042/ecma262-proposal-json-superset ).

**Comment by jfrazzano** [comment]
I agree.  And it seems like half the utility of an "app" in the first place.  Off line data manipulation with compossible navigation bars. No one needs to leave the client side, if authors reuse their dom bits, and have data maps for the next state of their apps.  No URIs  also makes blob objects, serverless client side apps, and the new browser "app" wrapper sort of silly. 

Its an app, but you have to be on a website to use it. Or change the view.  

It would be actually funny if liability under wai didn't shift completely off the vender and onto the author, as soon as the token "App" is applied  to what now seems like  less than last years  website. 

I was using imbedded data uris last September in a full  Polymer app.  Obviously component standards changed, so that work became irrelevant, but data uri was an option for the object element, up until recently.

 Similarly, last month at least and possibly last week, the object element could meet its requirements  as an embedded wrappper, simply by selecting "html" as its type.    Last night, no longer could find that in the spec.  And it said you now have to label the object element Application, Document, image or the other plug-in types.  I had this very conversation with some google devs in  the last two weeks. It was a little heartbreaking to see the language changed this morning.  More than that, it made tyhe announcement of the app wrapper just comic.

Without data, its not an app. Without off line and a record of state to handle your users; its not an app. 

As for the thin wrapper, when did that happen?   And as a number of videos of work i did last year show, you use the data attribute. Not href.  Andref did take the spec example from last week `/html`; 

My faith in this process the last two years, well... Lets just say I don't know whta kind of development is being advocated.
